2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -258,3 +261,68 @@ A causa dei rallentamenti incontrati, alcuni ruoli hanno visto un monte ore effe
 Il ruolo dell'amministratore invece, incaricato di gestire il reparto tecnologico implementando gli automatismi necessari, a causa di difficoltà incontrate nell'implementazione (errori, testing, verifica), ha visto un aumento delle ore rispetto a quanto preventivato.
 === Burndown-chart
 [da inserire]
+
+== Terzo Sprint dal 27-11-2023 al 04-12-2023
+
+=== Obiettivi raggiunti
+- proseguimento del processo di individuazione e stesura degli use case;
+- perfezionamento delle automazioni di versionamento documenti;
+- perfezionamento del template usato per i documenti;
+- aggiornamento parziale del documento di Norme di Progetto;
+- studio iniziale della libreria Three.js;
+- meeting con l'azienda Proponente per esporre ipotesi e use case individuati, richiedere chiarimenti ed avanzare oppurtune richieste;
+- contatto con l'azienda Proponente per fissare il prossimo meeting.
+
+=== Obiettivi mancati
+- aggiornamento completo del documento di Norme di Progetto con l'inserimento dei grafici di Gantt e di burndown.
+
+=== Analisi problematiche insorte
+
+Nonostante la realizzazione soddisfacente della maggior parte degli obiettivi concordati, sono emerse alcune criticità durante l'implementazione del progetto:
+
+- si è riscontrata una pianificazione non ottimale e superficiale, attribuibile alla mancanza di un dettagliato processo di pianificazione durante la riunione precedente l'avvio dello sprint;
+- problemi sono sorti a causa della mancata definizione di standard per la creazione dei grafici di Gantt e burndown, comportando una stesura parziale dei paragrafi nel documento del Piano di Progetto;
+- la durata dei meeting ha superato le aspettative a causa di alcune inefficienze temporali;
+- il team ha rilevato la mancanza di standard per designare le persone responsabili della redazione dei verbali durante lo sprint.
```

Comment:
Per questo problemi non sono state adottate soluzioni?

Reply:
Per quanto riguarda la redazione dei verbali non ricordo se siano stati fissati degli standard, per gli altri tre problemi le risoluzioni sono indicate nel paragrafo consecutivo, magari li posso formattare in elenco puntato per metterle più in risalto

Reply:
@AlessioBanzato @Carraro-Riccardo responsabili di oggi e di ieri, cosa ne dite? Quale risoluzione adottiamo per il problema dell'assegnazione del redattore dei verbali?

Reply:
Aggiornate con la decisione presa